src/pages/SignUp.tsx

```diff
@@ -7,6 +8,9 @@ import useFetch from '../hooks/useFetch';
 /**
  * This component is the page responsible for signing up.
  */
+
+// TODO: add Ionic alerts for invalid username/password
```

Comment:
A rough approach of this should have been implemented already.

Reply:
You're right I noticed that, it looks good. Forgot to remove the TODO.